src/main/java/net/apnic/whowas/nameserver/controller/NameServerController.java

```diff
@@ -0,0 +1,44 @@
+package net.apnic.whowas.nameserver.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Rest controller for the RDAP /nameserver path segment
+ *
+ * Controller is responsible for dealing with current state RDAP path segments.
+ * Currently implemented as a placeholder and returns HTTP Not Implemented.
+ */
+@RestController
+@RequestMapping("/nameserver")
+public class NameServerController
+{
+    private final static Logger LOGGER = LoggerFactory.getLogger(NameServerController.class);
+
+    /**
+     * GET request handler for nameserver path segment.
+     */
+    @RequestMapping(value="/{handle}", method=RequestMethod.GET)
+    @ResponseStatus(HttpStatus.NOT_IMPLEMENTED)
+    public void nameserverPathGet(@PathVariable("handle") String handle)
+    {
+        LOGGER.info("nameserver GET path query for {}", handle);
```

Comment:
Info logging is probably too high for a "not implemented" method. I'd recommend debug level for all these types of methods.